subsys/bluetooth/controller/ll_sw/ull_sched.c

```diff
@@ -170,33 +175,61 @@ int ull_sched_adv_aux_sync_free_slot_get(uint8_t user_id,
 #endif /* CONFIG_BT_CTLR_ADV_EXT && CONFIG_BT_BROADCASTER */
 
 #if defined(CONFIG_BT_CONN)
+#if defined(CONFIG_BT_CENTRAL)
 int ull_sched_after_cen_slot_get(uint8_t user_id, uint32_t ticks_slot_abs,
 				 uint32_t *ticks_anchor, uint32_t *us_offset)
 {
 	uint32_t ticks_to_expire;
 	uint32_t ticks_slot;
+	uint32_t remainder;
 	uint8_t ticker_id;
 
 	ticker_id = after_match_slot_get(user_id, ticks_slot_abs,
-					 ticker_match_op_cb, ull_hdr_get_cb,
+					 ticker_match_cen_op_cb, ull_hdr_get_cb,
 					 ticks_anchor, &ticks_to_expire,
-					 &ticks_slot);
+					 &remainder, &ticks_slot);
 	if (ticker_id != TICKER_NULL) {
-		*us_offset = HAL_TICKER_TICKS_TO_US(ticks_to_expire +
-						    ticks_slot) +
-						    (EVENT_JITTER_US << 3);
+		uint32_t central_spacing_us;
+		uint32_t remainder_us;
+		uint32_t slot_us;
+
+		/* When CONFIG_BT_CTLR_CENTRAL_SPACING is used then ticks_slot
+		 * returned converts to slot_us that is less than or equal to
+		 * CONFIG_BT_CTLR_CENTRAL_SPACING, because floor value in ticks
+		 * is returned as ticks_slot.
+		 * Hence, use CONFIG_BT_CTLR_CENTRAL_SPACING without margin.
+		 *
+		 * If actual ticks_slot in microseconds is larger than
+		 * CONFIG_BT_CTLR_CENTRAL_SPACING, then add margin between
+		 * central radio events.
+		 */
+		slot_us = HAL_TICKER_TICKS_TO_US(ticks_slot);
+		if (slot_us > CONFIG_BT_CTLR_CENTRAL_SPACING) {
+			central_spacing_us = slot_us;
+			central_spacing_us += (EVENT_TICKER_RES_MARGIN_US << 1);
```

Comment:
Many places we simply add `EVENT_TICKER_RES_MARGIN_US` and some places we add `EVENT_TICKER_RES_MARGIN_US << 1`. Can you explain why/when we do one or the other?

Reply:
The margin between two radio events that jitter is `EVENT_TICKER_RES_MARGIN_US << 1` and when inserting a new radio event between two existing event then the margin used while finding a free space is `EVENT_TICKER_RES_MARGIN_US << 2`.

There are only 3 occurrences of use of `EVENT_TICKER_RES_MARGIN_US` in ull_sched.c of above two types.